internal/function/uast.go

```diff
@@ -184,6 +202,39 @@ func (f UAST) Eval(ctx *sql.Context, row sql.Row) (out interface{}, err error) {
 		xpath = x.(string)
 	}
 
+	modeSet := false
```

Comment:
maybe is worth it to move this to a private function to improve readability?

Reply:
Is not used anymore as it is not an optional parameter.